scylladb/sphinx-scylladb-theme
# Fix shadow of topic with horizontal scroll

Closes #333 

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open https://sphinx-theme.scylladb.com/stable/examples/topic-box.html#topic-with-horizontal-scroll-mobile with your favorite browser

4. The topic box shadow should not be cut:

![image](https://user-images.githubusercontent.com/9107969/151126490-b1683a36-8a98-4b48-95d8-0dc053f07422.png)
